CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Contributing to GBIF Pipelines
+
+The GBIF Pipelines development community welcomes contributions from anyone! Thank you.
+
+If you have questions please open [an issue](https://github.com/gbif/pipelines/issues/new) or contact the developers at informatics@gbif.org.
```

Comment:
As Matt has suggested, perhaps we need a mailing list for GBIF and ALA devs.

Reply:
My thoughts were not to at this point. If we find it is used and there is a need to, we can create one at short notice and change this guide but I expect we won't. Initially, I expect we'd be able to guide people towards creating issues and keeping discussion there and joining slack instead.

Happy to create one now if others think it is necessary at this stage.

Reply:
Maybe a mailing list will give a better impression of building a community. It would also allow Living Atlas coordinators to follow conversations.

Reply:
pipelines@lists.gbif.org ?